magento/magento2
# Shipping Estimation: Prevent Quote Double Saving

### Description (*)
+ Removes a seemingly superfluous additional saving of the quote when estimating shipping
    + [The `Magento\Checkout\Model\Cart::save()` function also saves the quote in the same manner](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Checkout/Model/Cart.php#L593)
+ Only sets `collectShippingRates` once
    + [The `Magento\Checkout\Model\Cart::save()` function also sets this](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Checkout/Model/Cart.php#L591)

### Manual testing scenarios (*)
1. Install sample data
2. Add a product to the cart
3. Proceed to the cart page
4. Estimate shipping using valid address data
5. Ensure the available shipping methods are displayed
6. Ensure the selecting of a shipping method updates cart totals correctly

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#36648: Shipping Estimation: Prevent Quote Double Saving
